### DISCUSSION TITLE
Where is the api key stored in the file system

### DISCUSSION THREAD

**Discussion by Darkcast** [projectdiscovery/cvemap#66] [Q&A] [answered]
I was wondering where in the file system is the api key stored
thanks in advance

**Comment by ehsandeep** [comment, marked as answer]
`$HOME/.pdcp/credentials.yaml` from https://docs.projectdiscovery.io/cloud/free/advanced#advanced-integration-options

**Reply by Darkcast** [reply]
Thank you captain :)
